src/plugins/plugins.h

```diff
@@ -204,4 +204,24 @@ class drakvuf_plugins
 
 /***************************************************************************/
 
+unicode_string_t* read_wchar_array(vmi_instance_t vmi, const access_context_t* ctx, size_t length);
+void safe_free_unicode_str(unicode_string_t* us);
+size_t wchar_string_length(vmi_instance_t vmi, const access_context_t* ctx);
+unicode_string_t* read_wchar_string(vmi_instance_t vmi, const access_context_t* ctx);
```

Comment:
Adding these functions to `plugins.h` is not the correct place for them, they should go to `libdrakvuf`